js/foundation.accordion.js

```diff
@@ -165,16 +155,16 @@ class Accordion {
 
   /**
    * Closes the tab defined by `$target`.
-   * @param {jQuery} $target - Accordion tab to close.
+   * @param {jQuery} $target - Accordion tab to close (`.accordion-content`).
    * @fires Accordion#up
    * @function
    */
   up($target) {
     var $aunts = $target.parent().siblings(),
         _this = this;
-    var canClose = this.options.multiExpand ? $aunts.hasClass('is-active') : $target.parent().hasClass('is-active');
+    var canClose = $aunts.hasClass('is-active');
 
-    if(!this.options.allowAllClosed && !canClose) {
+    if(!this.options.allowAllClosed && !$aunts.hasClass('is-active')) {
```

Comment:
As far as I saw checking is not explicitly needed, but I guess checking none the less would be a good idea.
Regarding the variable - yes, I made it obsolete and not removed it.

I will fix both issues.